gemet/thesaurus/urls.py

```diff
@@ -92,6 +92,8 @@
             name='concept_edit'),
         url(r'^group/(?P<code>\d+)/edit$', edit_views.GroupEditView.as_view(),
             name='group_edit'),
+        url(r'^publish_version/$', edit_views.PublishVersionView.as_view(),
+            name='publish_version'),
```

Comment:
Move this url before `concept_add`.  Use a pattern consistent with the other patters (e.g. `/version/release`)